lib/ui/widgets/basewidget.js

```diff
@@ -79,4 +78,53 @@ BaseWidget.prototype.getHandledSources = function() {
   }];
 }
 
+BaseWidget.prototype.pushProps = function(widget, valueStream) {
```

Comment:
`pushProps` is not very descriptive - `updateStream`maybe?

Reply:
Yes, pushProps renamed in updateStream